src/main/java/com/conveyal/datatools/manager/models/FeedVersion.java

```diff
@@ -274,6 +274,12 @@ public void load(MonitorableJob.Status status, boolean isNewVersion) {
             assignGtfsFileAttributes(gtfsFile);
             String gtfsFilePath = gtfsFile.getPath();
             this.feedLoadResult = GTFS.load(gtfsFilePath, DataManager.GTFS_DATA_SOURCE);
+
+
+            FeedSource flexSetFeedSource = this.parentFeedSource();
+            flexSetFeedSource.flex = this.feedLoadResult.isGTFSFlex();
```

Comment:
@miles-grant-ibigroup is feedLoadResult guaranteed not to be null at this point? I had to add a check for this in `FeedSource#presetFlex`.

Reply:
I've made a corresponding gtfs-lib commit that always returns true or false https://github.com/conveyal/gtfs-lib/pull/335/commits/e4d5e611f14aceefc06676756c8f70e4c2180473

Reply:
@miles-grant-ibigroup It is still possible for `feedLoadResult` to be null and therefore fail when a call is made to `isGTFSFlex()` method.

Reply:
Thanks for looking into and fixing that!